iothub/service/src/ClientApiVersionHelper.cs

```diff
@@ -12,7 +12,7 @@ internal class ClientApiVersionHelper
     {
         private const string ApiVersionQueryPrefix = "api-version=";
         private const string ApiVersionGA = "2016-02-03";
-        private const string ApiVersionDefault = "2018-08-30-preview";
+        private const string ApiVersionDefault = "2020-05-31-preview";
         private const string ApiVersionLimitedAvailability = "2020-03-13";
```

Comment:
How are you verifying this works e2e?

Reply:
The E2E is running in the pipeline so it works in terms of existing scenarios. For new scenarios, as I mentioned in all other places, a test will be added when the service is in Canary.